pkg/cli/dev.go

```diff
@@ -0,0 +1,180 @@
+package cli
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"strings"
+	"time"
+
+	v1 "github.com/acorn-io/acorn/pkg/apis/api.acorn.io/v1"
+	cli "github.com/acorn-io/acorn/pkg/cli/builder"
+	"github.com/acorn-io/acorn/pkg/client"
+	"github.com/acorn-io/acorn/pkg/deployargs"
+	"github.com/acorn-io/acorn/pkg/dev"
+	"github.com/acorn-io/acorn/pkg/rulerequest"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+)
+
+func NewDev(c CommandContext) *cobra.Command {
+	cmd := cli.Command(&Dev{out: c.StdOut, client: c.ClientFactory}, cobra.Command{
+		Use:               "dev [flags] IMAGE|DIRECTORY [acorn args]",
```

Comment:
I don't think this bit indicates very well that you can attach to a running app. Maybe something like this?

```suggestion
		Use:               "dev [flags] IMAGE|DIRECTORY|APP [acorn args]",
```